libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -49,8 +49,7 @@ export class Bucket implements IBucketClient, ISimulatorResourceInstance {
     return this.context.withTrace({
       message: `Put Json (key=${key}).`,
       activity: async () => {
-        const filename = join(this.fileDir, key);
-        await fs.promises.writeFile(filename, JSON.stringify(body, null, 2));
+        await this.addFile(key, JSON.stringify(body, null, 2));
```

Comment:
There were already tests that were working for this refactor but they only tested adding single object so I added another test to add a bunch. but Im not sure how the console is doing it because I think the `bucket.list()` was always working